packages/core/test/acceptance/bootstrapping/application.feature.md

```diff
@@ -23,6 +23,5 @@ const app = new Application({
 });
 
 // get metadata about the registered components
-console.log(app.find('component.*')); // [Bindings] should match the 4 components registered above
```

Comment:
We probably need to introduce lifecycle support for components at later point. I think we should make components visible to the container.

Another benefit is that we can dump out/visualize bindings from the container to get a complete picture of all artifacts.